### DISCUSSION TITLE
slidesOffsetBefore and slidesOffsetAfter relatives units

### DISCUSSION THREAD

**Discussion by AdamReznicek** [nolimits4web/swiper#7138] [Q&A]
Hi,

i need create something like this, see picture below.
I have a container (width: 100%, max-width: 1640px) and i found slidesOffsetBefore, slidesOffsetAfter for offsets.
This works fine, but not with responsive design, cause slidesOffsetBefore and slidesOffsetAfter works with absolute units (px).

How can i create this effect and responsive?
<img width="693" alt="image" src="https://github.com/nolimits4web/swiper/assets/30239598/1ef3cee2-89e4-43e0-b5af-2a2ad5e8d5d1">

**Comment by AdamReznicek** [comment]
Solved:


`.section {
  overflow: hidden;
}

.container {
  max-width: 1640px;
}

.swiper {
  overflow: visible;
}

.swiper-slide {
  width: 500px;
}`

**Comment by MangoMarcus** [comment]
I have the exact same issue. The `.swiper { overflow: visible; }` solution does work well, but not for navigation or pagination as the `slidesPerView` doesn't actually match what's visible on screen

It would be useful to make `slidesOffsetBefore` a relative value or a CSS variable
